btsieve/src/bitcoin/mod.rs

```diff
@@ -1,13 +1,13 @@
 mod bitcoind_connector;
 mod blockchain_info_connector;
-mod queries;
+mod pattern_matching;
```

Comment:
IMO module names + the structs they define should match up, esp. if the modules are private and we re-export the type from the inside.

Reply:
Implemented as suggested.